spec/features/vehicles/index_spec.rb

```diff
@@ -2,9 +2,20 @@
 
 describe 'as a visitor when I visit the vehicle index page' do
   it 'Then I see the name of each vehicle record in the system' do
+    plane = Vehicle.create(name:"Boeing 747", locked: true, passenger_capacity: 100)
+    car = Vehicle.create(name:"Toyota Highlander", locked: false, passenger_capacity: 4)
+    truck = Vehicle.create(name:"Ford Raptor", locked: true, passenger_capacity: 5)
     visit '/vehicles'
     expect(page).to have_content("Boeing 747")
     expect(page).to have_content("Toyota Highlander")
     expect(page).to have_content("Ford Raptor")
   end
+
+  it 'Then I see a link which takes me to a creation page' do
+    visit '/vehicles'
+    expect(page).to have_link("creation link")
```

Comment:
From the user story: _Then I see a link to create a new Parent record, "New Parent"_
So I think New Vehicle should be the link text to match the story.

Reply:
creation link was the id, I'll use the link_to helper and search by name rather than id for clarity.